src/ploomber/util/default.py

```diff
@@ -257,6 +257,27 @@ def entry_point_relative(name=None):
     return location_pkg or location
 
 
+def try_to_find_env_yml(path_to_spec):
+    """
+
+    The purpose of this function is to check whether there are env files
+    ending with .yml format. It will return that file if it does exist,
+    otherwise return None.
+
+    This function will only be called right after path_to_env_from_spec.
+
+    """
+    # FIXME: delete this
```

Comment:
remove this comment

Reply:
let's ask @edublancas , why we don't need this check anymore?
# FIXME: delete this
    if path_to_spec is None:
        return None

Do we alert the user this path doesn't exist somewhere else? (and this is guaranteed to have some value if called).

Reply:
I wrote this code >1 a year ago 😂 so I don't fully remember. I think it's a case that will never happen anymore. but better to remove it and see if all tests pass to confirm

Reply:
I think i have tried to remove it, and it caused lots of tests to fail. I will confirm again.

Reply:
haha yeah I figured it's pretty old. Alright, we'll have to tackle it at some point, or just remove the fixme

Reply:
removed two fixme. ready to merge